definitions/tools/gather_to_sub_directory.cwl

```diff
@@ -0,0 +1,41 @@
+#! /usr/bin/env cwl-runner
+
+cwlVersion: v1.0
+class: CommandLineTool
+baseCommand: ["/bin/bash","directory_gatherer.sh"]
+
+requirements:
+    - class: ShellCommandRequirement
```

Comment:
Is this tool using the `ShellCommandRequirement`?  It doesn't look like it's taking advantage of any shell features to invoke the script.